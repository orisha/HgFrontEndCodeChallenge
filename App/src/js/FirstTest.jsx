/**
 * @author: Diego Favero <diego.g.favero@gmail.com>
 * @since : Jun 2020
 */
import React from "react";

/**
 * Generates the FirstTest Box
 */
class FirstTest extends React.Component {
    constructor(props) {
        super(props);
        this.state = {};

    }

    render(){

        return(

            <div> Say Hello Test !</div>

        )

    }
}
export default FirstTest
