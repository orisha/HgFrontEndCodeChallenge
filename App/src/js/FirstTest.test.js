import React from 'react';
import {default as Enzyme, shallow} from 'enzyme';
import FirstTest from './FirstTest.jsx';
import {describe, expect, it} from "@jest/globals";
import Adapter from 'enzyme-adapter-react-16';


Enzyme.configure({ adapter: new Adapter() })
describe("FirstTest", () => {
    it("should render initial layout", () => {
        // when
        const component = shallow(<FirstTest />);
        // then
        expect(component.getElements()).toMatchSnapshot();
    });
});
