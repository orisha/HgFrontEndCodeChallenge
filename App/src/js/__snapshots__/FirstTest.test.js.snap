// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FirstTest should render initial layout 1`] = `
Array [
  <div>
     Say Hello Test !
  </div>,
]
`;

exports[`Plan should render initial layout 1`] = `Array [<div> Say Hello Test !</div>,]`;
